### DISCUSSION TITLE
Move ~/lock directory?

### DISCUSSION THREAD

**Discussion by heapifyman** [Manjaro-Sway/manjaro-sway#549] [Q&A] [answered]
https://github.com/manjaro-sway/desktop-settings/commit/546b969f65fc813800d33406fb85f4fd1517b11b added a `lock` directory to `$HOME`.

It does not hurt but it looks a bit out of place.

Maybe move it to `$HOME/.local/state` or `$XDG_RUNTIME_DIR`?

**Comment by boredland** [comment, marked as answer]
Sure, would you like to make the relevant changes to https://github.com/manjaro-sway/desktop-settings/blob/sway/community/sway/usr/share/sway/scripts/once.sh ?

**Reply by heapifyman** [reply]
Ah you've changed it already. Thanks.

**Reply by boredland** [reply]
But I didn't even do a -git package release. Hope I will find time tomorrow
